public/javascripts/Gallery/src/filter/CityMenu.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * City Menu module.
+ * This is responsible for holding the buttons allowing users to filter citys by city type.
+ *
+ * @param uiCityMenu UI element corresponding to CityMenu.
+ * @returns {CityMenu}
+ * @constructor
+ */
+function CityMenu(uiCityMenu) {
+    let self = this;
+
+    let status = {
+        currentCityType: null
```

Comment:
You'll want to change the all the references to "city type" in this file to just "city", including in the comments.